cli/cmd/report_definitions.go

```diff
@@ -22,12 +22,18 @@ import (
 	"strings"
 
 	"github.com/lacework/go-sdk/api"
+	"github.com/lacework/go-sdk/internal/array"
 	"github.com/olekukonko/tablewriter"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var (
+	reportDefinitionsCmdState = struct {
+		// filter report definitions by subtype. 'Aws', 'GCP' or 'Azure'
+		SubType string
+	}{}
+
```

Comment:
Just a single string definition for this struct, is this for future expansion?

Reply:
Yeh, this is the pattern we use for flags. Though you are right it is not needed right now. I'll leave it here for any potential future flags